data-prepper-plugins/otel-trace-raw-processor/src/test/java/com/amazon/dataprepper/plugins/processor/oteltrace/OTelTraceRawProcessorTest.java

```diff
@@ -19,22 +20,30 @@
 public class OTelTraceRawProcessorTest {
 
     private final static ObjectMapper OBJECT_MAPPER = new ObjectMapper();
-    private final OTelTraceRawProcessor oTelTraceRawProcessor = new OTelTraceRawProcessor(new PluginSetting(null, Collections.EMPTY_MAP));
+    PluginSetting pluginSetting;
+    private OTelTraceRawProcessor oTelTraceRawProcessor;
+
+    @Before
+    public void setup() {
+        pluginSetting = new PluginSetting("OTelTrace", Collections.EMPTY_MAP);
+        pluginSetting.setPipelineName("OTelTraceRawProcessor");
+        oTelTraceRawProcessor = new OTelTraceRawProcessor(pluginSetting);
```

Comment:
You are not doing any assertion on the metrics here. Check the AbstractPrepperTest to see how you can verify that the correct metrics are being posted.

Reply:
Will add it